src/components/calcite-radio-button/calcite-radio-button.tsx

```diff
@@ -60,7 +60,7 @@ export class CalciteRadioButton {
 
   /**
    * The focused state of the radio button.
-   * @private
+   * @internal
```

Comment:
@eriklharper these should be marked as `internal` right? I see use of private but I don't think that does anything. Is that correct?

Reply:
yes, internal should suffice.